index.pug

```diff
@@ -0,0 +1,2366 @@
+          alt="Placeholder",
+          width="149",
+          height="87"
+        )
+    .slick-slide
+      .inner
+        img(
+          src="https://demo.dvio.global/lp/razorpay/pug/images/logos/11.png",
+          alt="Placeholder",
+          width="149",
+          height="87"
+        )
+    .slick-slide
+      .inner
+        img(
+          src="https://demo.dvio.global/lp/razorpay/pug/images/logos/12.png",
+          alt="Placeholder",
+          width="149",
+          height="87"
+        )
+// banner sec2 end
+// Business Type Section Start
+.container-fluid.sec3
+  p Choose the perfect solution for your business type
+  .container
+    .buttons
+      a(href="#institution") Educational Institutions
+    .buttons
+      a(href="#online-course") Online Courses
+    .buttons
+      a(href="#freelance") Freelance coaches &amp; tutors
+// Business Type Section End
+// Institutions Section Start
+#institution.container-fluid.sec4
+  .custom-shape-divider-top-1632475320
+    svg(
+      data-name="Layer 1",
+      xmlns="http://www.w3.org/2000/svg",
+      viewbox="0 0 1200 120",
+      preserveaspectratio="none"
+    )
+      path.shape-fill(d="M1200 120L0 16.48 0 0 1200 0 1200 120z")
+  .container
+    .Sec4TopLeft
+      h1 INSTITUTIONS
+      p
+        | Collect fees for your school, college or
+        br
+        |
+        | coaching class with or without a website
+    .Sec4TopRight
+      img(
+        src="https://demo.dvio.global/lp/razorpay/pug/images/institute.svg",
+        alt=""
+      )
+  .container
+    .box1.institution
+      img(
+        src="https://demo.dvio.global/lp/razorpay/pug/images/payment-button.svg",
+        alt=""
+      )
+      h2 Payment Links
+      ul
+        li Share links via Whatsapp, FB, Instagram
+        li Set automated reminders for collecting fees
+      a(href="#home") SIGN UP NOW
+    .box1.institution
+      img(
+        src="https://demo.dvio.global/lp/razorpay/pug/images/ic1.jpg",
+        alt=""
+      )
+      h2 Smart Collect
+      ul
+        li Create multiple free virtual accounts with Smart Collect
+        li Customise Convenience Fee bearer
+      a(href="#home") SIGN UP NOW
+    .box1.institution
+      img(
+        src="https://demo.dvio.global/lp/razorpay/pug/images/ic1.jpg",
+        alt=""
+      )
+      h2 Route
+      ul
+        li Automatically split &amp; transfer incoming payments to various internal accounts
+      a(href="#home") SIGN UP NOW
+  p.botText
+    | Transparent pricing that works for your business. No set up or maintenance fee.
+    a(href="#", target="_blank") Know More
+  .custom-shape-divider-bottom-1632475645
+    svg(
+      data-name="Layer 1",
+      xmlns="http://www.w3.org/2000/svg",
+      viewbox="0 0 1200 120",
+      preserveaspectratio="none"
+    )
+      path.shape-fill(d="M1200 120L0 16.48 0 0 1200 0 1200 120z")
+// Institutions Section End
+// Online Courses Section Start
+section#online-course
```

Comment:
use this code structure everywhere.

keep each section formatted like 

section#banner
   ........ Code
   
section#trusted
   ........ Code
   
section#qr-illustration
    ....... Code


make this change everywhere